bundles/org.openhab.binding.miio/README.md

```diff
@@ -55,7 +55,7 @@ Note. The Xiaomi devices change the token when inclusion is done. Hence if you g
 ## Binding Configuration
 
 No binding configuration is required. However to enable cloud functionality enter your Xiaomi username, password and server(s).
-The list of the known countries and related severs is [here](#country-servers).
+The list of the known countries and related servers is available [in the country servers](#country-servers) section.
```

Comment:
This should also be changed in the readme_base.md, as the miio readme is automatically generated from the readme base & the database files

Reply:
Thank you, adapted that file too.